tests/spec/core/issues-notes-spec.js

```diff
@@ -126,6 +126,51 @@ describe("Core — Issues and Notes", function () {
     );
   });
 
+  it("shows labels for github issues", async () => {
+    const githubConfig = {
+      github: "https://github.com/mock-company/mock-repository",
+      githubAPI: `${window.location.origin}/tests/data`,
+    };
+    const ops = {
+      config: githubConfig,
+      body:
+        makeDefaultBody() +
+        `
+        <div class='issue' data-number='1540'>issue is open on github</div>
+        <div class='issue' id='this-is-404' data-number='404'>this is 404</div>
+        <section id='issue-summary'></section>
+        `,
+    };
+    const doc = await makeRSDoc(ops);
+
+    const issueDiv = doc.getElementById("issue-1540");
+    expect(issueDiv).toBeTruthy();
+
+    const issueDiv404 = doc.getElementById("this-is-404");
+    expect(issueDiv404).toBeTruthy();
+    expect(issueDiv404.innerText).toEqual("this is 404");
```

Comment:
nit: `innerText` is a magic (it trims whitespaces for example) whereas `textContent` is (nearly) raw.